### ISSUE TITLE
Release 0.1 not working on server

### ISSUE THREAD

**Issue by nesnoj** [rl-institut/WAM#52]
I tried to update **WAMdev** to the latest release 0.1 which throws
```
...
Step 14/24 : RUN conda env create -f /tmp/environment.yml
 ---> Running in aece169230f6
Collecting package metadata: ...working... done
Solving environment: ...working... done

Downloading and Extracting Packages
certifi-2019.3.9     | 155 KB    | ########## | 100% 
readline-7.0         | 392 KB    | ########## | 100% 
setuptools-41.0.1    | 656 KB    | ########## | 100% 
zlib-1.2.11          | 120 KB    | ########## | 100% 
openssl-1.1.1b       | 4.0 MB    | ########## | 100% 
wheel-0.33.4         | 40 KB     | ########## | 100% 
sqlite-3.28.0        | 1.9 MB    | ########## | 100% 
ca-certificates-2019 | 126 KB    | ########## | 100% 
pip-19.1.1           | 1.9 MB    | ########## | 100% 
Preparing transaction: ...working... done
Verifying transaction: ...working... done
Executing transaction: ...working... done
ERROR: Could not open requirements file: [Errno 2] Datei oder Verzeichnis nicht gefunden: '/tmp/requirements.txt'

==> WARNING: A newer version of conda exists. <==
  current version: 4.6.1
  latest version: 4.6.14

Please update conda by running

    $ conda update -n base -c defaults conda

CondaValueError: pip returned an error

ERROR: Service 'wam' failed to build: The command '/bin/bash -c conda env create -f /tmp/environment.yml' returned a non-zero code: 1
```
It worked before at a8e6db75e9c20045c641162d97984817d911661b (for full changelist of release 0.1 since this commit see [here](https://github.com/rl-institut/WAM/compare/a8e6db75e9c20045c641162d97984817d911661b...release-0.1#diff-b7cb75ca59993a879e29bed8cd8469b8)).
@Bachibouzouk: In 8bf75947ca842c64d51934f44e33fb3850fac472 you moved the requirements to an extra txt file which is not found, apparently. No idea why.
@henhuy: Does the reqs file need to be copied somewhere manually?

BTW: I created a temporary branch as I need it urgently [`working_version_may2019`](https://github.com/rl-institut/WAM/tree/working_version_may2019). It does not harm any of our apps, just contains a specific commit I need.

**Comment by Bachibouzouk** [comment]
@nesnoj : I did that to avoid duplicates of requirements and to allow a user to install with pip or with conda. I propose a fix in #53

**Comment by nesnoj** [comment]
> allow a user to install with pip or with conda

Sounds reasonable!

I will test your fix right now.

**Comment by nesnoj** [comment]
Works out, thank you! I merged it.

I updated the WAMdev and will update WAMprod tonight.

**Comment by nesnoj** [comment]
Should we merge those fixes into 0.1?

**Comment by Bachibouzouk** [comment]
Now we make 0.1.1 ;)
We should also merge it in dev
